packages/publisher/src/generate-packages.ts

```diff
@@ -275,6 +275,14 @@ export function createReadme(typing: TypingsData): string {
   lines.push("# Details");
   lines.push(`Files were exported from ${definitelyTypedURL}/tree/${sourceBranch}/types/${typing.subDirectoryPath}.`);
 
+  if (typing.dtsFiles.length === 1 && packageFS.readFile(typing.dtsFiles[0]).length < 2500) {
+    const dts = typing.dtsFiles[0]
+    lines.push(`## ${typing.dtsFiles[0]}`)
+    lines.push("```ts")
+    lines.push(packageFS.readFile(dts))
```

Comment:
Technically breakable by multiline comments, but not worth guarding against 😆 

````ts
/**
 * @example
```
gotcha()!
```
 */
declare function gotcha(): void;
````